pyproject.toml

```diff
@@ -168,52 +171,30 @@ Documentation = "https://mne.tools/"
 Forum = "https://mne.discourse.group/"
 "Source Code" = "https://github.com/mne-tools/mne-python/"
 
-[build-system]
-requires = ["setuptools>=45", "setuptools_scm[toml]>=6.2", "wheel"]
-build-backend = "setuptools.build_meta"
-
-[tool.setuptools.packages.find]
-where = ["."]
-include = ["mne*"]
-namespaces = false
-
-[tool.setuptools_scm]
-version_scheme = "release-branch-semver"
-
-[tool.setuptools]
-include-package-data = true
-
-[tool.setuptools.package-data]
-"mne" = [
-    "data/eegbci_checksums.txt",
-    "data/*.sel",
-    "data/icos.fif.gz",
-    "data/coil_def*.dat",
-    "data/helmets/*.fif.gz",
-    "data/FreeSurferColorLUT.txt",
-    "data/image/*gif",
-    "data/image/*lout",
-    "data/fsaverage/*.fif",
-    "channels/data/layouts/*.lout",
-    "channels/data/layouts/*.lay",
-    "channels/data/montages/*.sfp",
-    "channels/data/montages/*.txt",
-    "channels/data/montages/*.elc",
-    "channels/data/neighbors/*.mat",
-    "datasets/sleep_physionet/SHA1SUMS",
-    "datasets/_fsaverage/*.txt",
-    "datasets/_infant/*.txt",
-    "datasets/_phantom/*.txt",
-    "html/*.js",
-    "html/*.css",
-    "html_templates/repr/*.jinja",
-    "html_templates/report/*.jinja",
-    "icons/*.svg",
-    "icons/*.png",
-    "io/artemis123/resources/*.csv",
-    "io/edf/gdf_encodes.txt",
+[tool.hatch.build]
+exclude = [
+    "/.*",
```

Comment:
this looks like it excludes all files at the root level of the repo; if that's right I don't see why we're explicitly excluding things like `Makefile` etc below. Am I misinterpreting `/.*` here?

Reply:
Overall I would recommend using inclusion overall rather than exclusion to account for new stuff in future

Reply:
> That is the same syntax as gitignore and matches any entry at the root that starts with a period

ah right, like glob not regex.

Reply:
~everything that's tracked in the VCS (git) is included by default, hence I picked the exclusion-based approach

Do you think that's not a good idea, @ofek?

Reply:
That's perfectly valid but my personal preference for source distributions is to only ship the project and test directories. Totally up to you!